README.md

```diff
@@ -0,0 +1,87 @@
+# PHP Finance - Double Entry
+
+[Double entry](https://en.wikipedia.org/wiki/Double-entry_bookkeeping) is an awesome proven approach to financial and analytical accounting. It allows doing accounting of any complexity.
+
+The library features are:
+
+- Generic double entry accounting implementation.
+- Multiple data storage backends available as separate packages.
+- Transactional.
+- Supports sub-accounts and dimensions.
+- Avoids precision problems.
+- Documentation and examples.
+
+## Glossary
+
+A short glossary is necessary for understanding what's going on.
+
+- **Asset** is something of value the company owns.
+- **Liability** is something that company owes to another company, bank or person.
+- **Debit** is an accounting entry that results in either increase of assets or decrease in liabilities.
+- **Credit** is an accounting entry that results in either decrease of assets or increase in liabilities.
```

Comment:
What increase/decrease depends on account.

Reply:
Yes and that is reflected in the description.